internal/apps/devflow/flow/flow.service.go

```diff
@@ -414,26 +626,27 @@ func (s *Service) GetDevFlowInfo(ctx context.Context, req *pb.GetDevFlowInfoRequ
 			extra := relationExtraMap[relationID]
 			mrInfo := relationMergeInfoMap[relationID]
 			appDto := appInfoMap[extra.AppID]
+			repoPath := gittarPrefixOpenApi + appDto.ProjectName + "/" + appDto.Name
```

Comment:
you should ensure `gittarPrefixOpenApi` has `/` at the end.

Reply:
done